### PR TITLE
Add silent and debug logging options

### LINKED ISSUE #152: Verbose mode
Is there a verbose mode? I don't see it documented. It would be really helpful to see the logs of all the things that Aragon calls.

### PR DESCRIPTION
**Pull request by jvluso**
resolves #152

### CODE REVIEW

**Comment by izqui** [comment]
@jvluso can you please rebase the branch with the current master? Due to merging #221 there are a bunch of conflicts

**Comment by jvluso** [comment]
I rebased it

File: src/commands/dao_cmds/upgrade.js [hunk 1]
```diff
@@ -47,16 +48,18 @@ exports.task = async ({ web3, reporter, dao, network, apmOptions, apmRepo, apmRe
         return setApp.send({ from: ctx.accounts[0], gasLimit: 1e6 })
       }
     }
-  ], { repo })
+  ], {
+    renderer: ListrRenderer(silent, debug)
```
Comment (0x6431346e) [inline src/commands/dao_cmds/upgrade.js]: This should be:
```
{
  repo,
  renderer: ListrRenderer(silent, debug)
}
```

**Comment by 0x6431346e** [comment]
We have a tiny problem now because debug messages are only displayed when `process.env.DEBUG` is truthy, which means we need to do `DEBUG=1 aragon cmd --debug`

**Comment by 0x6431346e** [comment]
Without `--debug`:
```
% dao acl 0x035ba8048C4Ab2d9478dFE3b56d09fE1c6538438
 ✔ Inspecting DAO Permissions
 ✔ Successfully fetched DAO apps for 0x035ba8048C4Ab2d9478dFE3b56d09fE1c6538438
 ✖ Cannot read property 'toString' of undefined
```
With `--debug` or `process.env.DEBUG` being truthy
``` 
 % dao acl 0x035ba8048C4Ab2d9478dFE3b56d09fE1c6538438 --debug
[14:49:48] Inspecting DAO Permissions [started]
[14:49:48] → Fetching permissions for 0x035ba8048C4Ab2d9478dFE3b56d09fE1c6538438...
[14:49:48] Inspecting DAO Permissions [completed]
 ✔ Successfully fetched DAO apps for 0x035ba8048C4Ab2d9478dFE3b56d09fE1c6538438
 ✖ Cannot read property 'toString' of undefined
 ❯ TypeError: Cannot read property 'toString' of undefined
    at /home/daniel/repos/aragon/aragon-cli/node_modules/cli-table/lib/index.js:182:27
    at Array.forEach (<anonymous>)
    at generateRow (/home/daniel/repos/aragon/aragon-cli/node_modules/cli-table/lib/index.js:181:11)
    at Table.toString (/home/daniel/repos/aragon/aragon-cli/node_modules/cli-table/lib/index.js:254:12)
    at /home/daniel/repos/aragon/aragon-cli/dist/commands/dao_cmds/acl_cmds/view.js:160:33
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
```

**Comment by 0x6431346e** [comment]
Removed the dates:
```
% aragon run --http localhost:8001 --http-served-from ./dist --debug
Start a local Ethereum network [started]
Starting a local chain from snapshot [started]
Local chain started at port 8545 [title changed]
Local chain started at port 8545 [completed]
Start a local Ethereum network [completed]
...
```

**Pull request merged**